### ISSUE TITLE
[BUG] Add Alt Text to Groups images 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5549]
**Describe the bug**
 This is a bug split from Epic https://github.com/rokwire/illinois-app/issues/5547. After consulting with Vanessa, it made sense to split the remediation effort by the affected app areas.

The scope of this bug covers:
Group cover photo uploaded by admin
Group posts with an image

**To Reproduce**

1. Enable Screen reader
2. Go to a 'Group' you are a member of that has a cover photo
3. Gesture through ensuring that image is announcing alt text to user
(Repeat for Group posts with an image)

**Expected behavior**
All non decorative images need alternative text to convey meaning to user.

**Actual behavior**
The images added before the Alt text field was created all have NULL alt text.

**Screenshots**

![Image](https://github.com/user-attachments/assets/09e17af9-d131-436c-a051-aea03e763efb)

 This alt text should work regardless of platform or device. Therefore not including info of specific test environments.

**Additional context**

**Comment by acovher2** [comment]
@TodorBachvarov Retested in 7.3.33 and passed for post and group pic. 

https://github.com/user-attachments/assets/96a5edbb-4ccc-40b3-98e3-fc7e6769e9c7

![Image](https://github.com/user-attachments/assets/f180e1c5-ad1d-4fc4-bc37-f068c2b02a39)

**Comment by acovher2** [comment]
@TodorBachvarov In 7.3.51 android the Group image is no longer being skipped as decorative and it also is missing alternative text. It need to be one or the other as it was in previous versions please.

**Comment by acovher2** [comment]
Verified as fixed in 8.0.8

**Comment by acovher2** [comment]
![Image](https://github.com/user-attachments/assets/17f7d31a-51c3-40a0-91af-e0b42a164f97)
